plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/FacetUninstallDelegate.java

```diff
@@ -41,4 +52,32 @@ private void updateClasspath(IProject project, IProgressMonitor monitor) throws
       javaProject.setRawClasspath(newClasspath, monitor);
     }
   }
+
+  /**
+   * Removes all the App Engine server runtimes from the list of targeted runtimes for
+   * <code>project</code>.
+   */
+  private void uninstallAppEngineRuntime(final IProject project, IProgressMonitor monitor) {
+    Job uninstallJob = new Job("Uninstall App Engine runtimes in " + project.getName()) {
```

Comment:
What's the reason to make this a Job but not the `removeAppEngineJarsFromClasspath` method's body?